Woof/Woof/Model/Owner/Owner.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+/**
+ A person who owns a pet (or several pets) and might be interested in finding a pet sitter for them.
+ */
+final class Owner: Identifiable, Codable {
+    /// Uniquely identifies the pet owner.
+    let id: UUID
+
+    /// The first name of the pet owner.
+    var name: String
+
+    /// The last name of the pet owner.
+    var surname: String
```

Comment:
nit:

I know the properties are the part of the owner:

```suggestion
    /// Unique identifier.
    let id: UUID

    /// The first name.
    var name: String

    /// The last name.
    var surname: String
```

and so on.